emails/tests/views_tests.py

```diff
@@ -1511,8 +1511,10 @@ class SnsInboundViewSimpleTests(SimpleTestCase):
     def setUp(self):
         self.valid_message = EMAIL_SNS_BODIES["s3_stored"]
         self.client = Client(
-            HTTP_X_AMZ_SNS_TOPIC_ARN=self.valid_message["TopicArn"],
-            HTTP_X_AMZ_SNS_MESSAGE_TYPE=self.valid_message["Type"],
+            headers={
+                "x-amz-sns-topic-arn": self.valid_message["TopicArn"],
+                "x-amz-sns-message-type": self.valid_message["Type"],
+            }
```

Comment:
praise: oh this is so much nicer to have `headers` dictionary instead of the awkward `HTTP_` variable names.